Tests/AvatarTests.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  AvatarTests.swift
+//  MessageKit
+//
+//  Created by Dan Leonard on 8/4/17.
+//  Copyright © 2017 Hexed Bits. All rights reserved.
+//
+
+import XCTest
+@testable import MessageKit
+
+class AvatarTests: XCTestCase {
+    
+    override func setUp() {
+        super.setUp()
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+    }
+    
+    func testNoParams() {
+        let avatar = Avatar()
+        XCTAssertEqual(avatar.initals, "?")
+        XCTAssertEqual(avatar.initalsLabel.text, "?")
+        XCTAssertEqual(avatar.layer.cornerRadius, 0.0)
```

Comment:
Looks like this is failing the tests on iOS 9+. Avatar has a default size of `30 x 30` resulting in a `cornerRadius` of `15.0`